source/renderer/app/components/staking/info/StakingInfo.js

```diff
@@ -1,15 +1,95 @@
 // @flow
 import React, { Component } from 'react';
 import { observer } from 'mobx-react';
+import { defineMessages, intlShape, FormattedMessage } from 'react-intl';
+import { Button } from 'react-polymorph/lib/components/Button';
+import { ButtonSkin } from 'react-polymorph/lib/skins/simple/ButtonSkin';
 import styles from './StakingInfo.scss';
 
-type Props = {
-  name: string,
-};
+const messages = defineMessages({
+  heading: {
+    id: 'staking.info.heading',
+    defaultMessage: '!!!Cardano is transitioning to a decentralized system',
+    description: 'Headline for the Decentralization progress notification.',
+  },
+  description: {
+    id: 'staking.info.description',
+    defaultMessage:
+      '!!!Cardano is transitioning from a federated to a decentralized system. During this transition, some new blocks will be produced by nodes operated in a federated way and some by nodes run by ada stakeholders. All rewards will be distributed to ada stakeholders and none to operators of federated nodes.',
+    description:
+      'Info description for the Decentralization progress notification.',
+  },
+  percentage: {
+    id: 'staking.info.percentage',
+    defaultMessage:
+      '!!!Currently, {percentage}% of the system is decentralized and operated by stake pools.',
+    description:
+      'Percentage info description for the Decentralization progress notification.',
+  },
+  buttonLabel: {
+    id: 'staking.info.buttonLabel',
+    defaultMessage: '!!!Learn more',
+    description: 'Button Label for the Decentralization progress notification.',
+  },
+});
+
+type Props = { percentage: number };
 
 @observer
-export default class StakingInfo extends Component<Props> {
+export default class DelegationInfo extends Component<Props> {
```

Comment:
If you're renaming the component to "DelegationInfo" the file name should also follow it.